servicemesh-install/topics/install-multi-tenant.adoc

```diff
@@ -0,0 +1,73 @@
+[[multi-tenant-install]]
+= Multi-tenant {ProductName} installation
+
+The {ProductName} operator provides support for multi-tenant control plane installations. A multi-tenant control plane is configured so that only specified namespaces can be joined into its {ProductShortName}, isolating the mesh from other installations.
+
+[NOTE]
+====
+You cannot use multi-tenant control plane installations in conjunction with a cluster-wide control plane installation. {ProductName} installations must either be multi-tenant or single, cluster-wide installations.
+====
+
+== Known issues
+
+* `MeshPolicy` is still a cluster-scoped resource and applies to all control planes installed in OpenShift. This can prevent the installation of multiple control planes or cause unknown behavior if one control plane is deleted
+* The Jaeger agent runs as a `DaemonSet`, therefore `tracing` may only be enabled for a single `ServiceMeshControlPlane` instance.
+* Automatic route creation is currently incompatible with multi-tenant {ProductShortName} installations. Ensure that it is disabled in your `ServiceMeshControlPlane` if you plan to attempt a multi-tenant installation.
+* If you Delete the project that contains the control plane before you delete the `ServiceMeshControlPlane` resource, some patrs of the installation may not be removed:
+** Service accounts added to the `SecurityContextConstraints` may not be removed
+** `OAuthClient` resources associated with Kiali may not be removed, or its list of redirectURIs may not be accurate
+
+== Comparison of multi-tenant and cluster-wide installations
+
+The main difference between a multi-tenant installation and a cluster-wide installation is the scope of privileges used by the control plane deployments, for example, galley and pilot. The components no longer use cluster-scoped Role Based Access Control (RBAC) `ClusterRoleBinding`, but rely on namespace-scoped RBAC `RoleBinding`. Every namespace in the `members` list will have a `RoleBinding` for each service account associated with a control plane deployment and each control plane deployment will only watch those member namespaces. Each member namespace has a `maistra.io/member-of` label added to it, where the `member-of` value is the namespace containing the control plane installation.
+
+== Configuring a multi-tenant installation
+
+You can configure a multi-tenant installation by setting the `multitenant` option to `true` in the istio section of the `ServiceMeshControlPlane` resource. For example,
+
+.Multi-tenant custom resource example
```

Comment:
You describe the yaml here... does it make sense to have an oc create...?
Also, we have shorthand:
 ServiceMeshControlPlane -- smcp
and ServiceMeshMemberRoll -- smmr

Should we mention these?

Reply:
@rcernich and @knrc can you address these comments/questions from @brian-avery?

Reply:
I'm not sure.  The actual "installation" is no different from a cluster-wide installation, it's just that the contents of the ServiceMeshControlPlane resource is different, in that it has `multitenant: true`.

Reply:
I don't think we need to include the oc command as this is really documenting additional options within the SMCP.  I also think we can ignore the short names since, IIRC, those should only be usable via the command line and not the resources themselves.

Reply:
Reviewed without changes.